### PR TITLE
Add action and reducer that changes the state when answering a question

### PR DESCRIPTION
**Pull request by Hacklor**
This does not go to the next question yet

### CODE REVIEW

File: src/reducers/progress.js [hunk 1]
```diff
@@ -1,4 +1,21 @@
...
+        answers: [
+          ...state.answers,
+          {
+            questionId,
+            answerId
+          }
+        ]
+      })
+      // Goto next question
+      return newState;
```
Comment (pascalw) [inline src/reducers/progress.js]: Undefined variable

File: src/reducers/progress.js [hunk 2]
```diff
@@ -1,4 +1,21 @@
 export default function progress(state = [], action) {
-  console.log(state, action);
+  switch(action.type) {
+    case 'ANSWER_QUESTION':
+      const questionId = action.questionId
+      const answerId = action.answerId
+
+      return Object.assign({}, state, {
```
Comment (pascalw) [inline src/reducers/progress.js:7]: Your initial state is an Array, but your computed state is an Object?

Comment (Hacklor) [reply src/reducers/progress.js:7]: The state is an hash from the key `progress` that contains the `currentQuestion` and an array of `answers`. I think it's correct

File: src/reducers/progress.js (hunk 1 above)
Comment (Hacklor) [reply src/reducers/progress.js]: Removed

**Pull request merged**